.compose.env

```diff
@@ -0,0 +1,6 @@
+# Determines how the dev environment should run. Set to insights for developing
```

Comment:
Minor: Naming may lead to confusion. The filename is a shell script, not a common `.env` format.